docs/scripts/gen_test_case_reference.py

```diff
@@ -58,19 +66,24 @@
 
 runner = CliRunner()
 logger.info(
-    f"Generating documentation for test cases until {GENERATE_UNTIL_FORK} as fill {' '.join(args)}"
+    f"Generating documentation for test cases until {GENERATE_UNTIL_FORK} as \
+    fill {' '.join(args)}"
```

Comment:
```suggestion
    f"Generating documentation for test cases until {GENERATE_UNTIL_FORK} as "
    f"fill {' '.join(args)}"
```

Adjacent strings concatenate in Python as has less murky whitespace handling.